.github/workflows/main.yml

```diff
@@ -194,19 +195,19 @@ jobs:
             dotnet: '3.1.x'
             java: '8'
             node: '10'
-            python: '3.6'
+            python: '3.8'
```

Comment:
Does this coincide with us raising our minimum Python requirements?

If not, how are we sure what we vend continues to support Python 3.6?

Reply:
This was actually me trying to understand if the intermittent pip install failures were caused by "old" versions of pip (as provided by venv). But it si not, and this will be reverted (I'd have sworn I already did!).

And so this only changed the default version of Python, but we'd still have tested with 3.6 elsewhere (maybe on a different OS, though).